internal/pkg/cli/svc_package.go

```diff
@@ -150,6 +160,34 @@ func newWorkloadStackGenerator(o *packageSvcOpts) (workloadStackGenerator, error
 	return deployer, nil
 }
 
+func newTemplateOverrider(pathToOverridesDir, app, env string, fs afero.Fs, provider *sessions.Provider) (clideploy.Overrider, error) {
+	info, err := override.Lookup(pathToOverridesDir, fs)
+	if err != nil {
+		var errNotExist *override.ErrNotExist
+		if errors.As(err, &errNotExist) {
+			return new(override.Noop), nil
+		}
+		return nil, fmt.Errorf("look up overrider at %q: %w", pathToOverridesDir, err)
+	}
+	switch {
+	case info.IsCDK():
+		provider.UserAgentExtras("override cdk")
```

Comment:
Maybe `cdk override` instead?

Reply:
I kept it this is way to hint at a hierharchy "override <option>" but 🤷 I can change if we feel strongly against it